### PR TITLE
Docs: Breaking change re: add_host

### PR DESCRIPTION
**Pull request by jaredbeck**
Sorry for nit-picking, hope it's helpful

[ci skip]

### CODE REVIEW

**Comment by mattbrictson** [comment]
I appreciate the grammar nitpicks (really!), but in this case the `add_host` method still *does* modify existing hosts in some scenarios. So the "will always add a new host" phrasing is actually incorrect.

If you can think of a better way to phrase the subtlety of the change, I'm all ears. :)

Here is a discussion of the change in question: https://github.com/capistrano/capistrano/pull/1846#issuecomment-279442350

**Comment by jaredbeck** [comment]
> .. the "will always add a new host" phrasing is actually incorrect.

Ah, thanks!

> If you can think of a better way to phrase the subtlety of the change, I'm all ears. :)

I have tried again, thanks.

**Comment by jaredbeck** [comment]
PS: I slightly prefer the phrase "minor breaking change" to "potentially breaking change". I find it more clear about that fact that a breaking change has been identified, while still explaining why it only warranted a minor version bump. However, I thought that wording might be controversial, and didn't want it to hold up this PR. So, do with that (rambling thought) as you will 😄

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,7 +20,11 @@ https://github.com/capistrano/capistrano/compare/v3.7.2...v3.8.0
 
 ### Potentially breaking changes:
```
Comment (mattbrictson) [inline CHANGELOG.md]: I'm amenable to changing this to "Minor breaking changes" 👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,7 +20,11 @@ https://github.com/capistrano/capistrano/compare/v3.7.2...v3.8.0
...
 
-* [#1846](https://github.com/capistrano/capistrano/pull/1846): add_host will add a new host in a case where it used to incorrectly update an existing one (potentially breaking) [(@dbenamy)](https://github.com/dbenamy)
+* [#1846](https://github.com/capistrano/capistrano/pull/1846): add_host -
+  When this method has already been called once for a given host and it is
+  called a second time with a port, a new host will be added. Previously,
+  the first host would have been updated.
+  [(@dbenamy)](https://github.com/dbenamy)
```
Comment (mattbrictson) [inline CHANGELOG.md]: Remove newlines, please?

Comment (jaredbeck) [inline CHANGELOG.md]: The newlines guarantee that all text editors will be able to read the changelog. Some text editors cannot read long lines. Are you sure you cannot accept hard wrapping?

Comment (mattbrictson) [inline CHANGELOG.md]: I'd still prefer no hard wrap. Maintainer's prerogative 😄

**Comment by jaredbeck** [comment]
> I'd still prefer no hard wrap. Maintainer's prerogative 😄

Changes complete.

**Comment by mattbrictson** [comment]
Thanks!! 🙇

**Pull request merged**